gitlab-pipeline/stage/test.yml

```diff
@@ -427,14 +427,14 @@ test_accep_raspberrypi4:
     reports:
       junit: results_accep_raspberrypi4.xml
 
-test_backend_integration_enterprise:
+test:integration:open_source:backend:
```

Comment:
I think the hierarchy here is weird, enterprise/open_source should be the last prefix, as this is the one we override in the next job.

I suggest `test:backend-integration:open_source/enterprise`. In my eyes, this suite is independent of `integration` so they shouldn't share that prefix neither.

Reply:
By the way, thank you very much for:
1) modify notation to use `:`
2) make enterprise extend open source and not the opposite.

Reply:
`test:backend-integration:open_source/enterprise` I agree. Much better, thanks!